manifests/init.pp

```diff
@@ -79,6 +79,7 @@
   $workers                  = 8,
   $mistral_api_url          = undef,
   $mistral_api_port         = '8989',
+  $mistral_api_service      = true,
```

Comment:
Default: `false` for release so behavior does not change, and then we can stress and get it into the hands of early users.